### PR TITLE
Fix indentation of the workflow

### PR DESCRIPTION
**Pull request by hug-dev**
Signed-off-by: Hugues de Valon <hugues.devalon@arm.com>

### CODE REVIEW

**Comment by sbailey-arm** [comment]
Should we specify a version of Rust to use, or is sticking to the latest OK?

**Comment by hug-dev** [comment]
Rust is stable, but I think sticking to a specific version is better in case new lints that might break CI are added. In my opinion, it is better to increase the version manually and fix all the problems in the same PR.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -10,10 +10,10 @@ jobs:
...
       - name: Install latest Rust
         uses: actions-rs/toolchain@v1
-          with:
-            profile: minimal
-            override: true
-            toolchain: stable
+        with:
+          profile: minimal
+          override: true
+          toolchain: stable
```
Comment (sbailey-arm) [inline .github/workflows/ci.yml:16]: `toolchain: 1.46.0`

**Review by sbailey-arm — CHANGES_REQUESTED** [review]
>Rust is stable, but I think sticking to a specific version is better in case new lints that might break CI are added. In my opinion, it is better to increase the version manually and fix all the problems in the same PR.
Agreed

File: .github/workflows/ci.yml (hunk 1 above)
Comment (sbailey-arm) [inline .github/workflows/ci.yml:16]:
```suggestion
          toolchain: 1.46.0
```

**Review by sbailey-arm — CHANGES_REQUESTED** [review]

**Review by ionut-arm — APPROVED** [review]

**Review by sbailey-arm — APPROVED** [review]

**Pull request merged**